src/main/resources/org/biouno/unochoice/DynamicReferenceParameter/index.jelly

```diff
@@ -62,5 +65,9 @@
     </j:forEach>
 
     UnoChoice.renderDynamicRenderParameter('#${paramName}', '${h.escape(it.getName())}', '${h.escape(paramName)}', referencedParameters, dynamicReferenceParameter);
+
+    // update spinner id
+    let paramId = document.querySelector('#${paramName}').querySelector('div').getAttribute('id').split('_').pop()
+    document.querySelector('#${paramName}-spinner').setAttribute('id', paramId + '-spinner')
```

Comment:
Hi @kinow,
It's because in dynamic reference parameters the paramName value is changed by the subelement (input, orderedList, formattedHtml,... ex : [here](https://github.com/jenkinsci/active-choices-plugin/blob/master/src/main/resources/org/biouno/unochoice/DynamicReferenceParameter/formattedHtml.jelly#L7) )
As this is this element which trigger the update method, I must change the id